portfolio/src/main/webapp/script.js

```diff
@@ -155,3 +155,30 @@ function updateMovieText(data) {
 
   return unorderedList;
 }
+
+function displayName(){
+  fetch('/data')
+  .then((response) => response.text()
+    
+    // //parse it as an html instead of text
+    // const parser = new DOMParser();
+	  // const htmlDoc = parser.parseFromString(response.text, 'text/html');
+    // console.log(htmlDoc);
+
+    // const introduction = htmlDoc.getElementsByTagName("h1")[0];
+
+    // console.log(introduction.textContent);
+
+    // return introduction.textContent;
+
+    // return response.text();
+  )
+  .then((introduction) => {
+    const parser = new DOMParser();
```

Comment:
In general, it probably wouldn't make sense to structure your api this way -- theres no need to encode the response in html if you are just gonna parse the data out, but you'll learn about JSON in the next lesson which would be the better form to structure data in, so this is fine for now

Reply:
Okay great!